src/containers/Body.module.css

```diff
@@ -0,0 +1,8 @@
+.dates {
+  box-sizing: border-box;
+  display: flex;
+  padding: 0px 30px;
+  padding-bottom: 30px;
+  /* justify-content: space-evenly; */
+  gap: 100px;

```

Comment:
Prova ad utilizzare `justify-content: space-between` invece di `gap`. In questa maniera gli elementi verranno disposti su tutta la larghezza del container a partire dai margini,